### ISSUE TITLE
Question about the new folder that has shown up?

### ISSUE THREAD

**Issue by ignaciogc** [Oak-Harbor-Kits/Starter-Kit-V2#1]
Did you mean to create a sub-folder with the same Starter-Kit-V2 content, or was it just a mistake with the latest update?
I feel like you meant to update the main files, but instead created a sub-folder.

**Comment by Oak-Harbor-Kits** [comment]
Yeah I think that was a mistake when I tried uploading a new version. Working on trying to remove it. Sorry about that
